app/controllers/users_controller.rb

```diff
@@ -9,7 +9,7 @@ class UsersController < ApplicationController
   after_action       :update_activity_time, :only => :login
 
   def index
-    @users = User.authorized(:view_users).except_hidden.search_for(params[:search], :order => params[:order]).includes(:auth_source, :cached_usergroups).paginate(:page => params[:page])
+    @users = User.authorized(:view_users).except_hidden.search_for(params[:search], :order => params[:order]).includes(:auth_source, :cached_usergroups).paginate(:page => params[:page]).includes(:cached_usergroups)
```

Comment:
This is already including `:cached_usergroups` - see the  method call before `paginate`